src/runtime_src/core/tools/common/ProgressBar.cpp

```diff
@@ -55,8 +55,8 @@ static boost::format fmtBatchPF("\n[%s]: %s < %s >\n");
 
 // ------ S T A T I C   F U N C T I O N S -------------------------------------
 
-static std::string
-format_time(std::chrono::duration<double> duration) 
+std::string
+ProgressBar::formatTime(std::chrono::duration<double> duration) 

```

Comment:
since this is no longer a static fucntion, should it go under a "member function" comment?

Reply:
It is still a static function for the header file was updated as well to be a static value.

Ultimately, it needs to go into the XBUtility, but didn't go to that in this release (it was getting fairly large).  In my next PR, it will be moved.